Tavisca.Bootcamp.LanguageBasics.Exercise2/ForumPostEasy.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise2
+{
+    class ForumPostEasy
```

Comment:
The class should be public. check the problem statement. This class should be static as it contains only one static method.